conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -46,6 +48,7 @@
 from apache_beam.io.gcp import bigquery #WriteToBigQuery
 
 THERESHOLD = 10485760 
+my_timeout = 5
```

Comment:
I have a hunch that nothing is going wrong, and that it's just that some 7z files are a lot bigger than others. Instead of a timeout that kills the sub-process, a risky messy business, lets do a print every 1000 revisions with the file and the revision number being processed. That way you can watch the log and get a meaningful understanding of where things are at, and if they are indeed broken in some way.

Reply:
Saw this after added a timeout. 
- I have a hunch that nothing is going wrong, and that it's just that some 7z files are a lot bigger than others -> This is also my guess.

But a suspicious thing was that there was no elements outputed to WriteToBigQuery when the ingestion job getting stuck.

The idea for timeout is just to ingest as many as possible than come back to the exceptionally large ones.

Reply:
Tested the timeout locally and remotely, made sure it can successfully stop the subprocess, record a job status and output the last processed revision number, also record processed revision results.